setup.py

```diff
@@ -13,6 +13,7 @@
     package_data={'': ['pybb/templates', 'pybb/static']},
     zip_safe=False,
     install_requires=[
+        'Django>=1.8',
```

Comment:
fixed. Thx. I'll create another PR for this change. PyBB requires Django>=1.8 so it should be in the setup.py.